mobile/Router.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Created by yura on 14.03.17.
+ */
+import React from 'react';
+import {Actions, Scene, Router} from 'react-native-router-flux';
+import SignIn from './SignIn';
+import SignUp from './SignUp';
+const scenes = Actions.create(
+    <Scene key="root">
+        <Scene key="signin"
+               component={SignIn}
+               title="SignIn"
+               initial />
+        <Scene
+            key="signup"
+            component={SignUp}
+            title="SignUp" />
+    </Scene>
+);
+/* ... */
+class _Router extends React.Component {
```

Comment:
Try to use as few symbols as you can. And rename your component to something else.
```
const Routing = () => <Router scenes={scenes} />;
Routing.displayName = 'Routing';
export default Routing;
```